### DISCUSSION TITLE
Migrate 7.9 -> 8.2

### DISCUSSION THREAD

**Discussion by michaelschefczyk** [LMS-Community/slimserver#757] [General]
Dear All,

For many years, I am running a self-built docker images of slimserver 7.9. That does work, however, I would like to upgrade to a current 8.x version. Whenever I do this with my own build or with the official container downloaded from docker hub, it fails. After starting the container, scanning music files does break somewhere in the middle. The console then has a lot of messages like:
Slim::Formats::Playlists::CUE::parse (503) Error: Couldn't get duration of filename here file:
Slim::Formats::Playlists::CUE::parse (522) Error: Missing file or start/end points for track file:
Can someone please be so kind to inform me what this may mean - and ideally how to resolve it?

Regards,

Michael Schefczyk

**Comment by mherger** [comment]
This is not a Docker issue, but a change in how LMS handles CUE sheets. It's more strict than before. Please check the CUE sheets for the affected files.

**Comment by michaelschefczyk** [comment]
Thank you very much for the quick response!

Unfortunately, my knowledge about CUE sheets is "slim". The general discussion seems to focus on .cue-files (https://forums.slimdevices.com/showthread.php?114063-Squeezebox-server-8-2-single-flac-image-play).

I mount my music files and my configuration files via NFS to the docker container. My music directory consists of folders and .flac-files. There are no .cue-files. I do use Kid3-qt on Linux to edit metadata. I am happy that both kodi and slim (until 7.9) can handle this. Is this what the system views as a problem? Is there a way to address the problem?

**Reply by michaelherger** [reply]
My recommendation would be to join the forum and ask the guys who worked on that change (and responded in the above discussion).
